packages/strapi-plugin-upload/admin/src/components/FiltersCard/index.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import Wrapper from './Wrapper';
+
+const FiltersCard = () => {
+  return <Wrapper type="button">Filters Form</Wrapper>;
```

Comment:
Your wrapper is a `div ` why don't you use a button?

Reply:
Sorry, I didn't want you both to review this files, I was not explicite enough in my description
`(Filters to be improved in another PR)` but it will be updated in the PR https://github.com/strapi/strapi/pull/5333/files